From: Markus Koschany <apo@debian.org>
Date: Wed, 11 Nov 2015 19:53:35 +0100
Subject: ivy 2.4.0
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: base64

Rml4IEZUQkZTIHdpdGggaXZ5IDIuNC4wLiBQYXRjaCB0YWtlbiBmcm9tCmh0dHBzOi8vZ2l0aHVi
LmNvbS9taXpkZWJzay9ncmFkbGUvY29tbWl0L2UwMTM4YjEKVGhhbmtzIHRvIE1pa2/FgmFqIEl6
ZGVic2tpCgpGb3J3YXJkZWQ6IG5vCg==
---
 .../ivyservice/ivyresolve/parser/IvyXmlModuleDescriptorParser.java      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/parser/IvyXmlModuleDescriptorParser.java
+++ b/subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/parser/IvyXmlModuleDescriptorParser.java
@@ -1207,7 +1207,7 @@
                 buffer = null;
                 state = State.INFO;
             } else if (state == State.EXTRA_INFO) {
-                getMd().getExtraInfo().put(new NamespaceId(uri, localName), buffer == null ? "" : buffer.toString());
+                ((Map)getMd().getExtraInfo()).put(new NamespaceId(uri, localName), buffer == null ? "" : buffer.toString());
                 buffer = null;
                 state = State.INFO;
             } else if (state == State.DESCRIPTION) {
